chapters/external-repository-identifiers.md

```diff
@@ -238,15 +238,24 @@ Examples:
 
 External Reference Site: [Software Heritage persistent IDentifiers](https://docs.softwareheritage.org/devel/swh-model/persistent-identifiers.html)
 
-### F.4.1 gitbom <a name="gitbom"></a>
+### F.4.2 gitbom <a name="gitbom"></a>
 
-Locator Format: gitoid":"\<git object type>":"\<hash algorithm>":"\<hash value>
-	
-Contextual Example: gitoid:blob:sha1:261eeb9e9f8b2b4b0d119366dda99c6fd7d35c64
+Locator Format: `"gitoid:"\<git object type>":"\<hash algorithm>":"\<hash value>`
+
+Locator Format Reference: <https://www.iana.org/assignments/uri-schemes/prov/gitoid>
+
+Current valid values are:
+* `<git object type>`: `blob`, `tree`, `commit`, `tag`
+* `<hash algorithm>`: `sha1`, `sha256`
+* `<hash value>` should be expressed as a hexadecimal string in lower case
+
+Contextual Examples:
+* `gitoid:blob:sha1:261eeb9e9f8b2b4b0d119366dda99c6fd7d35c64` is the gitbom identifier of a software artifact using the SHA1 algorithm
+* `gitoid:bom:sha256:3557f7eb43c621c71483743d4b37059bb80933e7f71277c0c3b3846159d1f61c` is the gitbom identifier of an artifact dependency graph using the SHA256 algorithm
```

Comment:
This example has an object type of `bom` which is not one of the `<git object type>` listed above (`blob`, `tree`, `commit`, `tag`)

Reply:
Thoughts on :

`gitbom:gitoid:blob:sha256:3557f7eb43c621c71483743d4b37059bb80933e7f71277c0c3b3846159d1f61c`